communities/all/resources/test_tools.json

```diff
@@ -406,6 +410,7 @@
             "Metagenomics"
         ],
         "ToolShed id": "spades",
+        "Date of first commit of the suite": "2024-03-12",
```

Comment:
The date here sounds wrong: https://github.com/galaxyproject/tools-iuc/commits/main/tools/spades

Reply:
its because the test runs on the dedicated test repo: https://github.com/paulzierep/Galaxy-Tool-Metadata-Extractor-Test-Wrapper - and there its the first commit (since I only copied some tools there), I checked